src/utils/math_utils.cairo

```diff
@@ -166,3 +167,36 @@ fn mod_subtraction(a: u256, b: u256) -> u256 {
         a - b
     }
 }
+
+impl ContractAddressPartialOrd of PartialOrd<ContractAddress> {
```

Comment:
What do you think of creating a separate `utils` file for this? I don't think ContractAddress should be considered part of math utils.